About/history.html

```diff
@@ -164,12 +164,12 @@
 	was recognised with numerous <a href="../Research/awards.html">awards</a>.
       </p>
       <p>
-        seL4 was first ported to new architectures and platforms during HACMS.
-        The original proof stack only held for Arm 32-bit hardware. During the
-        project the proof was ported to the Intel x64 architecture. This was the
-        start of a continuing widening of the scope of <a
-        href="../todo.html">all supported architectures and platforms that seL4
-        now supports.</a>
+        The HACMS program also marked the beginning of porting seL4 to new
+        architectures and platforms. The original proof stack only held for Arm
+        32-bit hardware. During the project the proof was ported to the Intel
+        x64 architecture. This was the start of a continuing widening of the
+        scope of <a href="../todo.html">all supported architectures and
+        platforms that seL4 now supports.</a>
```

Comment:
Let's merge it, but I'll probably want to fiddle with this paragraph a bit more in the future.